OpenMTRDemo/Forms/ExpandedImageDialog.cs

```diff
@@ -108,31 +121,42 @@ private void SetDisableableControls(bool state)
         private void cannyThreshold1Number_ValueChanged(object sender, EventArgs e)
```

Comment:
Since we are already in here how about we consolidate event handlers similar to the double click handler I made in TiledFilterForm?